ads/google/a4a/utils.js

```diff
@@ -183,6 +190,10 @@ export function googleBlockParameters(a4a, opt_experimentIds) {
   if (opt_experimentIds) {
     eids = mergeExperimentIds(opt_experimentIds, eids);
   }
+  if (new RegExp(`(^|,)${ADX_ADY_EXP.experiment}($|,)`).test(eids)) {
+    slotRect.left = slotRect.left || 1;
```

Comment:
Will these ever be falsey for other reasons (like couldn't measure)?

Reply:
Hidden slots never send an ad request so no should never be falsey.  If you like, I can explicitly check === 0.

Reply:
Looking through the construction of this, seems like we shouldn't generate negative numbers so 0 should be the only case anyway. LGTM.